### ISSUE TITLE
Bug with line chart API

### ISSUE THREAD

**Issue by ORESoftware** [GoogleWebComponents/google-chart#210]
I am seeing what appears to be a bug with the line chart API:
https://developers.google.com/chart/interactive/docs/gallery/linechart

here is the problem:

https://stackoverflow.com/questions/49122990/getting-cannot-read-property-vb-of-undefined-error-with-google-chart-api

the error appears to be internal to the Chart API lib:

<img width="331" alt="screenshot 2018-03-05 19 48 38" src="https://user-images.githubusercontent.com/11139560/37013414-9cd54044-20ae-11e8-987b-1371c60ca1ea.png">

**Comment by ORESoftware** [comment]
Yeah this is a bug, this happens when Google Charts receives an empty array:

`data.addRows([]);`

that's something that Google Charts should handle, instead of breaking. :(

**Comment by wesalvaro** [comment]
You're right that there isn't a whole lot of error checking in cases like that.
It'd be a good idea to not only handle that but also add some tests for these kinds of cases.

**Comment by ORESoftware** [comment]
+1

**Comment by AndreMendesMarcondes** [comment]
You should wait for finish the request, put await or then after the request and wait it to end
